composer/modules/js-tests/src/test/resources/failing/natives-9.bal

```diff
@@ -4,63 +4,94 @@ package ballerina.io;
 @Return {value:"Fields listed in the record"}
-public native function <TextRecordChannel channel> readTextRecord()(string []);
+public native function <DelimitedRecordChannel channel> nextTextRecord () (string[]);
 
 @Description {value:"Function to write text records"}
-@Param {value:"records: feilds which are included in the record"}
-public native function <TextRecordChannel channel> writeTextRecord(string [] records);
+@Param {value:"channel: The DelimitedRecordChannel to write records"}
+@Param {value:"records: Fields which are included in the record"}
+public native function <DelimitedRecordChannel channel> writeTextRecord (string[] records);
 
 @Description{value:"Function to close the text record channel"}
-public native function <TextRecordChannel channel> closeTextRecordChannel();
+@Param {value:"channel: The DelimitedRecordChannel to be closed"}
+public native function <DelimitedRecordChannel channel> closeDelimitedRecordChannel();
 
 @Description {value:"Function to read characters"}
-@Param {value:"numberOfChars: number of characters which should be read"}
-@Return {value:"the characters which will be read"}
+@Param {value:"channel: The CharacterChannel to read characters from"}
+@Param {value:"numberOfChars: Number of characters which should be read"}
+@Return {value:"The character sequence which was read"}
 public native function <CharacterChannel channel> readCharacters(int numberOfChars)(string);
 
+@Description {value:"Function to read all characters in the give I/O source"}
+@Return {value:"all characters read"}
+public native function <CharacterChannel channel> readAllCharacters()(string);
+
 @Description {value:"Function to write characters"}
-@Param {value:"content: text content which should be written"}
-@Param {value:"startOffset: if the content needs to be written with an offset"}
-@Return {value:"int: number of characters written"}
+@Param {value:"channel: The CharacterChannel to write characters to"}
+@Param {value:"content: Text content which should be written"}
+@Param {value:"startOffset: If the content needs to be written with an offset, the value of that offset"}
+@Return { value:"Number of characters written"}
 public native function <CharacterChannel channel> writeCharacters(string content,int startOffset)(int);
 
-@Description {value:"Function to close character channel"}
+@Description {value:"Function to close a character channel"}
+@Param {value:"channel: The CharacterChannel to be closed"}
 public native function <CharacterChannel channel> closeCharacterChannel();
 
 @Description { value:"Function to read bytes"}
-@Param { value:"numberOfBytes: number of bytes which should be read" }
-@Return { value:"blob: the bytes which were read" }
-@Return {value:"int: number of bytes which were read"}
+@Param {value:"channel: The ByteChannel to read bytes from"}
+@Param { value:"numberOfBytes: Number of bytes which should be read" }
+@Return { value:"The bytes which were read" }
```

Comment:
Shall we reformat this file. There are several different conventions followed now. (i.e space after { etc)